packages/kbn-scout/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/scout
+
+Empty package generated by @kbn/generate
```

Comment:
Maybe we can add here some initial information about the purpose of this package etc?

Reply:
updated in 12832cf7c67b6075113b5c7677757af724293216